qiskit/circuit/library/generalized_gates/isometry.py

```diff
@@ -139,8 +139,8 @@ def _gates_to_uncompute(self):
         Call to create a circuit with gates that take the desired isometry to the first 2^m columns
          of the 2^n*2^n identity matrix (see https://arxiv.org/abs/1501.06911)
         """
-        q = QuantumRegister(self.num_qubits)
-        circuit = QuantumCircuit(q)
+        q = QuantumRegister(self.num_qubits, "q")
+        circuit = QuantumCircuit(q, name="isometry")
```

Comment:
This is not the full circuit, correct? Should we possibly call it something like "isometry_to_uncompute"?

Reply:
Thanks, I've updated the name.
I also removed all the old synthesis code from the statepreparation class.